backend/src/main/scala/cromwell/backend/wdl/WriteFunctions.scala

```diff
@@ -27,8 +29,10 @@ trait WriteFunctions extends IoFunctionSet with AsyncIoFunctions {
     }
   }
 
+  def qualifyPath(pathFrom: String): String = if (new File(pathFrom).isAbsolute) pathFrom else  System.getProperty("user.dir") + "/" + pathFrom
```

Comment:
Could possible replace this method with `File.currentWorkingDirectory.relativize(File("/your/absolute/path"))`?

Reply:
Wouldn't that go from absolute -> relative when you intend to go relative -> absolute?

Reply:
Oh I see.  Then you could standardize on relative paths.